### DISCUSSION TITLE
AtlasMap can't get target document from .adm file imported into java

### DISCUSSION THREAD

**Discussion by YukiYasha** [atlasmap/atlasmap#4335] [Q&A] [answered]
I imported the target xml file into the AtlasMap Data Mapper UI as below:
`<?xml version="1.0" encoding="UTF-8"?>
<ns:XmlOE xmlns:ns="http://atlasmap.io/xml/test/v2" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
    <ns:Address>
        <ns:addressLine1>1040 Notexisting St</ns:addressLine1>
        <ns:zipCode>01886</ns:zipCode>
    </ns:Address>
    <ns:Contact>
        <ns:fullName>Totton</ns:fullName>
        <ns:phoneNumber>123-456-7890</ns:phoneNumber>
        <ns:zipCode>01886</ns:zipCode>
    </ns:Contact>
</ns:XmlOE>`

Then I exported it to *.adm file and after that importing it to the eclipse. There is the log: 
![image](https://user-images.githubusercontent.com/43841617/189863017-1deae521-0007-4ee5-82ee-a7c3d9a7bb78.png)

I run main class and get errors. If I use *.adm file from example project and re-export it from AtlasMap Data Mapper UI, it work well.

**Comment by igarashitm** [comment, marked as answer]
You'd need to copy `Document ID`of the imported target document from UI and use it as a key to retrieve a target document. Or simply use `getDefaultTargetDocumentIO`. When you import a document, an auto generated `Document ID` is assigned which has GUID suffix. So the target document is no longer `XMLInstanceSource`.

**Reply by YukiYasha** [reply]
Thank you so much. `getDefaultTargetDocument()` works well in my case.

Just to be more sure, I believe it is the `Document ID` you refer to.
![image](https://user-images.githubusercontent.com/43841617/190047370-3db82588-06a2-4ad8-b17d-35b3d1a785af.png)
